.github/workflows/publish.yaml

```diff
@@ -0,0 +1,26 @@
+name: publish
+on:
+  push:
+    branches:
+      - main
+  schedule:
+    - cron:  '0 0 * * *'
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+          submodules: true
+      - name: Install Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.x
+      - name: Install Python dependencies
+        run: pip install -r requirements.txt
+      - name: Install PlantUML
+        run: sudo apt-get -y install plantuml
```

Comment:
Installing plantuml is pretty heavy. It depends on Java & lots of libraries that do not come preinstalled on the ubuntu worker.

Reply:
maybe we can use https://kroki.io#install , there is a free service available, or I check what is required to spin up an instance, if there is a way to make it not too public

Reply:
PS: this is of course not a thing for now, but for some future ...